runtime/Go/antlr/interval_set.go

```diff
@@ -23,6 +23,18 @@ func NewInterval(start, stop int) *Interval {
 	return i
 }
 
+func (i *Interval) GetStart() int {
+	return i.start
+}
+
+func (i *Interval) GetStop() int {
+	return i.stop
+}
+
+func (i *Interval) Contains(item int) bool {
+	return i.contains(item)
```

Comment:
Better to update the code calling contains() to call Contains() instead of having two versions.  Since contains() is internal to the package it should be rather easy to do.